caffe2/python/operator_test/stats_ops_test.py

```diff
@@ -11,20 +11,37 @@
 class TestCounterOps(TestCase):
 
     def test_stats_ops(self):
-        workspace.FeedBlob('k', np.array(['k0', 'k1'], dtype=str))
-        workspace.FeedBlob('v', np.array([34, 45], dtype=np.int64))
+        # The global StatRegistry isn't reset when the workspace is reset,
+        #   so there may be existing stats from a previous test
+        workspace.RunOperatorOnce(core.CreateOperator(
+            'StatRegistryExport', [], ['prev_k', 'prev_v', 'prev_ts']))
+        previous_keys = workspace.FetchBlob('prev_k')
+        existing = len(previous_keys)
+
+        keys = [__name__ + '/key1', __name__ + '/key2']
```

Comment:
what if it is from the same file? Shouldn't this include the test name? You could also use line number.